builtin/logical/rabbitmq/backend.go

```diff
@@ -0,0 +1,119 @@
+package rabbitmq
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+	"github.com/michaelklishin/rabbit-hole"
+)
+
+// Factory creates and configures Backends
+func Factory(conf *logical.BackendConfig) (logical.Backend, error) {
+	return Backend().Setup(conf)
+}
+
+// Backend creates a new Backend
+func Backend() *framework.Backend {
+	var b backend
+	b.Backend = &framework.Backend{
+		Help: strings.TrimSpace(backendHelp),
+
+		PathsSpecial: &logical.Paths{
+			Root: []string{
```

Comment:
We've generally moved away from root paths as the ACL system is very flexible and allows `sudo` to be given on top of other sets of capabilities. So you can remove this and just let the ACL system do its job. :-)